### DISCUSSION TITLE
Can't instal 3.15.4-1_arm64 on Debian 6.6.62-1

### DISCUSSION THREAD

**Discussion by chris-lambert-shiels** [GrandOrgue/grandorgue#2082] [Help]
Fails with "unmet dependencies":

_grandorgue:
              Depends: libwxbase3.0-0v5 (>= 3.0.4+dfsg) but it is not installable
              Depends: libwxgtk3.0-gtk3-0v5 (>= 3.0.4+dfsg) but it is not installable
              Depends: libwxgtk3.0-gtk3-0v5 (>= 3.0.4+dfsg-10~) but it is not installable
E: Unable to correct problems, you have held broken packages._

I interpret this to mean that it depends on earlier versions of those libraries.

Not a problem - but I thought I'd mention it.

**Comment by oleg68** [comment]
- Seems your are trying to install arm64 instead of amd64.
